packages/backend/src/app.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { RouterModule } from '@nestjs/core';
+import { ApiV1Module } from './v1/api.module';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: process.env.NODE_ENV === 'development' ? '.env.development.template' : '.env.production',
+      isGlobal: true,
+    }),
+    ApiV1Module,
+    RouterModule.register([
```

Comment:
The @Controller() accepts a path `@Controller("/api/v1")` which may be more maintainable since then we don't need to use the RouterModule at all.

Reply:
yeah we will need to use the router module when react app is merged though